contributing/code/stack_trace.rst

```diff
@@ -0,0 +1,182 @@
+Getting a Stack Trace
+=====================
+
+When :doc:`reporting a bug </contributing/code/bugs>` for an
+exception, it is instrumental that you provide one or several stack
```

Comment:
```suggestion
exception, it is relevant that you provide one or several stack
```
?

Reply:
"instrumental" might be too strong, but "relevant" is too weak. I'll use "crucial"

Reply:
and "a wrong code behavior" seems incorrect, I'll use "a wrong behavior in code"